lib/tasks/web_errors.rake

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+require 'json-schema'
+require 'erubis'
+
+namespace :slack do
+  namespace :web do
+    namespace :api do
+      desc 'Update Web API errors from official spec'
+      task :update_errors do
+        spec = JSON.parse(open('https://raw.githubusercontent.com/slackapi/slack-api-specs/master/web-api/slack_web_openapi_v2.json').read)
+        known_typos = %w[
```

Comment:
Oh wow. Open a ticket with Slack for this or an issue in their repo?

Reply:
I opened an issue (https://github.com/slackapi/slack-api-specs/issues/34), but I also just asked @episod about this [on Slack](https://dev4slack.slack.com/archives/C0K8617D5/p1584118044350200?thread_ts=1584114323.343900&cid=C0K8617D5). He thinks it's a typo in the underlying metadata that feeds into the spec, but that the typo would not exist in the actual API responses. I wasn't sure before whether it was the right choice to omit these, but I think this new information validates the omission.

Reply:
Yes of course, I'm just saying that we need to let Slack know, which you did. Thanks.